src/main/java/io/strimzi/kafka/metrics/YammerMetricsCollector.java

```diff
@@ -58,27 +59,27 @@ public List<MetricFamilySamples> collect() {
                 Metric metric = entry.getValue();
                 LOG.trace("Collecting Yammer metric {}", metricName);
 
-                String name = metricName(metricName);
+                String prometheusMetricName = metricName(metricName);
                 // TODO Filtering should take labels into account
-                if (!config.isAllowed(name)) {
-                    LOG.info("Yammer metric {} is not allowed", name);
+                if (!config.isAllowed(prometheusMetricName)) {
+                    LOG.info("Yammer metric {} is not allowed", prometheusMetricName);
                     continue;
                 }
-                LOG.info("Yammer metric {} is allowed", name);
+                LOG.info("Yammer metric {} is allowed", prometheusMetricName);
                 Map<String, String> labels = labelsFromScope(metricName.getScope());
                 LOG.info("labels " + labels);
```

Comment:
Use placeholders in logging patterns. Two reasons why this is best practice:
- avoids pointless garbage.. in this case the runtime needs to be call map.toString() and create a new String object containing the concatenation `"labels " + labels.toString()`.   All of that is pointless garbage/cpu overhead if logging is turned down/off.
- what if the `labels`, when stringified contains the characters `{}`?  How do you know what the underlying log implementation will do?



```suggestion
                LOG.info("Yammer metric {} is allowed", prometheusMetricName);
                Map<String, String> labels = labelsFromScope(metricName.getScope());
                LOG.info("labels {} ", labels);
```

Reply:
Thanks for the explanation. Done